src/ape/utils/basemodel.py

```diff
@@ -74,7 +76,36 @@ def provider(self) -> "ProviderAPI":
             raise ProviderNotConnectedError()
         return self.network_manager.active_provider
 
-    def create_contract_container(self, contract_type: "ContractType") -> "ContractContainer":
+    def create_contract_type(self, address: AddressType) -> Optional[ContractType]:
```

Comment:
The name being kind of weird (maybe `get_contract_type` makes more sense)

Reply:
```suggestion
    def get_contract_type(self, address: Union[str, AddressType]) -> Optional[ContractType]:
```

And then add this to the body:
```py
converted_address: AddressType = conversion_manager.convert(address, AddressType)
```